.Lib9c.Tests/Action/ValidatorDelegation/AllocateRewardTest.cs

```diff
@@ -130,7 +130,7 @@ public void Execute_WithoutReward_Throw()
     [Theory]
     [InlineData(0)]
     [InlineData(1181126949)]
-    [InlineData(793705868)]
+    [InlineData(707058493)]
```

Comment:
Why should we change random seed values?

Reply:
> Why should we change random seed values?

I'll add a new seed.